src/famfs_cli.c

```diff
@@ -1368,6 +1368,90 @@ do_famfs_cli_verify(int argc, char *argv[])
 	return 0;
 }
 
+/********************************************************************/
+void
+famfs_flush_usage(int   argc,
+	    char *argv[])
+{
+	char *progname = argv[0];
+
+	printf("\n"
+	       "famfs flush: Flush or invalidate the processor cache for an entire file\n"
+	       "\n"
+	       "This command is useful fur shared memory that is not cache coherent. It should\n"
```

Comment:
nitpick:

fur -> for
also in famfs-cli-reference.md

Reply:
The cli reference is auto-generated via scripts/generate-cli-reference.sh, which is why it's in both places 🗡️ 

Fixing...